src/MetalsFeatures.ts

```diff
@@ -15,10 +15,9 @@ export class MetalsFeatures implements StaticFeature {
     if (!params.capabilities.experimental) {
       params.capabilities.experimental = {};
     }
-    params.capabilities.experimental.debuggingProvider = false;
-    params.capabilities.experimental.decorationProvider = workspace.isNvim
-      ? true
-      : false;
+    (params.capabilities.experimental as any).debuggingProvider = false;
```

Comment:
So I noticed before that it was warning that `debuggingProvider` wasn't a member of object before. Is doing this `params.capabilities.experimental as any` basically just turning off the typing checking here?

Reply:
Exactly. Experimental is typed as `object` so there’s no much else we can do here, since it’s inherently dynamic